filebeat/module/system/auth/ingest/pipeline.json

```diff
@@ -54,8 +54,13 @@
         {
             "date": {
                 "if": "ctx.event.timezone != null",
-                "field": "@timestamp",
-                "formats": ["ISO8601"],
+                "field": "system.auth.timestamp",
+                "target_field": "@timestamp",
+                "formats": [
+                    "MMM  d HH:mm:ss",
```

Comment:
There is a test failing because it has a timestamp like `Apr 4 03:39:57`, that would need a pattern like this one, but with a space less (`MMM d HH:mm:ss`), could you add it to these lists of formats?

Reply:
Sure